tests/acceptance/features/coreApiSharePublicLink2/multilinkSharing.feature

```diff
@@ -1,3 +1,4 @@
+@skipOnReva
```

Comment:
You can also remove `@skipOnRevaMaster` tag and merge the example tables.

Reply:
Imo it's better if done on another PR